src/lava/magma/compiler/builders/py_builder.py

```diff
@@ -7,15 +7,18 @@
 import numpy as np
 from lava.magma.compiler.builders.interfaces import AbstractProcessBuilder
 
-from lava.magma.compiler.channels.interfaces import AbstractCspPort
-from lava.magma.compiler.channels.pypychannel import CspRecvPort, CspSendPort
+from message_infrastructure import (
+    AbstractTransferPort,
+    RecvPort,
+    SendPort
+)
 from lava.magma.compiler.utils import (PortInitializer, VarInitializer,
                                        VarPortInitializer)
 from lava.magma.core.model.py.model import AbstractPyProcessModel
-from lava.magma.core.model.py.ports import (AbstractPyIOPort,
-                                            IdentityTransformer, PyInPort,
-                                            PyOutPort, PyRefPort, PyVarPort,
-                                            VirtualPortTransformer)
+from message_infrastructure.ports import (AbstractPyIOPort,
```

Comment:
Is it possible to bring it under the lava.message_infrastructure umbrella? Please check this with @awintel regarding convention.

Reply:
Yes. Need to do following steps:
- move the code from `src/lava/magma/runtime/message_infrastructure` to `src/lava/message_infrastructure`.
- remove the sub-project message_infrastructure and build cpp message_infrastructure under lava potery.